src/System.Net.Http/src/System/Net/Http/Managed/HttpConnection.cs

```diff
@@ -338,14 +339,14 @@ private Task WriteFormattedInt32Async(int value, CancellationToken cancellationT
                     _readLength = bytesRead;
                 }
 
-                // The request is no longer retryable; either we received data from the _readAheadTask, 
+                // The request is no longer retryable; either we received data from the _readAheadTask,
```

Comment:
If possible, try to avoid whitespace changes like this, it makes the review more difficult and messes with history.